### ISSUE TITLE
Add a guide for alt text

### ISSUE THREAD

**Issue by dmundra** [CivicActions/accessibility#726]
Are guide is lacking some directions for alt text, we could write a few paragraphs and then link to resources:

- https://www.tpgi.com/what-is-alt-text-and-how-to-write-it/
- https://design.va.gov/content-style-guide/alternative-text-for-images
- https://webaim.org/techniques/alttext/
- https://www.w3.org/WAI/tutorials/images/decision-tree/

**Comment by MarcusMorba** [comment]
Thanks. Will start to prepare a draft after sept. 8th. Please stay patient.
